pombola/core/models.py

```diff
@@ -784,7 +784,38 @@ def get_absolute_url(self):
         return ('organisation_kind', (self.slug,))
 
 
+COMMITTEE_SLUGS = [
+    'ad-hoc-committees',
+    'joint-committees',
+    'national-assembly-committees',
+    'ncop-committees',
+]
+
 class OrganisationQuerySet(models.query.GeoQuerySet):
+    def committees(self):
+        return self.filter(kind__slug__in=COMMITTEE_SLUGS)
+
+    def ongoing(self):
+        """
+        Filter committees with start dates before the current date and end dates
+        either null, '', 'future' or after the current date.
+        """
+        now = datetime.date.today()
+        now_approx = ApproximateDate(year=now.year, month=now.month, day=now.day)
+        return self.filter(Q(started__lte=now_approx) & (
+            Q(ended='') | Q(ended='future') | 
+            Q(ended__isnull=True) | Q(ended__gte=now_approx)
+        ))
+    
+    def has_email_contacts(self):
+        return self.filter(contacts__kind__slug='email')
+
+    def contactable(self):
```

Comment:
```suggestion
    def contactable_committees(self):
```
since it assumes committees